modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/WatchProcessor.java

```diff
@@ -158,7 +158,7 @@ private static CompletableFuture<Void> notifyWatches(List<WatchAndEvents> watchA
 
             try {
                 notifyWatchFuture = watchAndEvents.events.isEmpty()
-                        ? watchAndEvents.watch.onRevisionUpdated(revision)
+                        ? completedFuture(null)
```

Comment:
Why isn't `onRevisionUpdated() invoked anymore?

Reply:
In reality, no one should use it after this commit, but we leave the code to get rid of it later, in a separate issue